rct229/rulesets/ashrae9012019/section22/section22rule19.py

```diff
@@ -38,7 +38,7 @@ def __init__(self):
             each_rule=Section22Rule19.HeatRejectionRule(),
             index_rmd=BASELINE_0,
             id="22-19",
-            description="The tower shall be controlled to maintain a leaving water temperature, where weather permits.",
+            description="The baseline heat rejection device shall be controlled to maintain a constant leaving water temperature.",
```

Comment:
Perhaps add back in where  weather permits since it is the language in the standard?

Reply:
if we include it in the rule description then I think it needs to be included in the rule somehow

Reply:
Decided to go with Jackson's suggested change per call on 8/14/2024